### PR TITLE
Update go example code

### PR DESCRIPTION
**Pull request by youshy**
# Description

Fix test examples

## Issues Resolved

n/a

## Check List

- [X] All tests pass. See TESTING.md for details.
- [X] New functionality includes testing.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ This file is used to list changes made in each version of golang.
 
 ## Unreleased
 
+## 5.1.1 - *2021-06-03*
```
Comment (Xorima) [inline CHANGELOG.md:7]:
```suggestion
```

**Review by Xorima — DISMISSED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,9 @@ This file is used to list changes made in each version of golang.
 
 ## Unreleased
 
+
```
Comment (ramereth) [inline CHANGELOG.md:7]: Can you please address this MDL error?

**Review by ramereth — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (Xorima) [inline CHANGELOG.md:7]:
```suggestion
```

**Review by Xorima — DISMISSED** [review]

**Review by Xorima — APPROVED** [review]

**Comment by ramereth** [comment]
@Xorima should we be concerned about the failure in `ubuntu-1804-src` ?

**Review by Xorima — APPROVED** [review]

**Comment by kitchen-porter** [comment]
Released as: [5.2.0](https://github.com/sous-chefs/golang/releases/tag/5.2.0)

**Pull request merged**